internal/gemini/renderer.go

```diff
@@ -42,6 +42,11 @@ var (
 	emphDelimiter      = []byte("*")
 	strongDelimiter    = []byte("**")
 	delDelimiter       = []byte("~~")
+	horizontalRule     = []byte("---")
```

Comment:
I'm genuinely interested if there are clients which would already render horizontal lines in a special way. For now, at least Lagrange doesn't seem to do that (does the text just below the first header get displayed as a subtitle or something?):

![Lagrange horizontal line test](https://user-images.githubusercontent.com/26599554/131206616-6f63176d-02dd-40cd-b4ad-5bdd684416ba.png)

Reply:
I don't know of any clients that present special formatting, but if you strip out a semantically meaningful divider then you change the meaning of the text. For example, in some books you see a simple `***` used to separate different unrelated passages, and the reader clearly understands that the symbols separate one section from another. Taking out that divider would cause confusion.